### ISSUE TITLE
HUD Not Dismiss

### ISSUE THREAD

**Issue by zjTek** [mutualmobile/MMProgressHUD#43]
While calling HUD frequently,I got a problem that the UI Jammed,the HUD stayed even I have called dismissMethod.And I'm sure I 'm running the latest version.
![29a3b539-dce1-4d56-99c6-dc8a2546b9e4](https://cloud.githubusercontent.com/assets/17873233/18118232/255d50b8-6f86-11e6-9a91-b950e0fbc1e4.png)

**Comment by larsacus** [comment]
It looks like you're using the demo app. What steps did you perform to reproduce this?

**Comment by zjTek** [comment]
Yep, I got this problem first on my App which integrates your code(thanks),It do fine on IOS9 but have little problem on IOS8 ,I guess there is some problem with my code But I found the same issue on your demo code while randomly click different HUD, and also this happens on IOS8 iphone4s

发自我的 iPhone
